resources/assets/uad_lists.json

```
@@ -1134,6 +1134,14 @@
     "neededBy": [],
     "labels": [],
     "removal": "Recommended"
+  },
+    "com.longcheertel.secretcode": {
+    "list": "Oem",
+    "description": "Refers to the system package that handles secret codes. \nSecret codes are special sequences of numbers and symbols that can be entered on a phone's keypad to access hidden features or information.",
```

```suggestion
    "description": "Refers to the system package that handles secret codes.\nSecret codes are special sequences of numbers and symbols that can be entered on a phone's keypad to access hidden features or information.",
```

This suggestion removes a redundant space. `\n` inserts a whiteline.